Docker file + docker-compose configuration for automatic deployment

luco5826 opened the pull request:
Solves issue #32 
It is possible to fully delpoy the entire website by typing
```
docker-compose up
```
### Main points:
- The firts build will take some time. 
- Every directory in the project is mounted as a volume, so that if something changes there's no need to re-build the image.
- Nginx and php-fpm configuration files are mounted too, for the same reason as above
- Right now the `weeehire.db` file is re-created at every container launch, see TODO n.3

### TODO:
1. Add TLS ecnryption to Nginx webserver
2. Url rewriting to hide the php extension
3. Determine how and when the `weeehire.db` is created in order to persist betwee launches

lvps commented on Dockerfile:
```diff
@@ -0,0 +1,45 @@
+RUN apt update
+RUN apt install -y\
+	sqlite3 \
+	gettext \
```
Not needed, we're using a PHP implementation of gettext asd

lvps replied:
I mean just the line that says "gettext", GitHub shows a 4 lines snippet for no reason

luco5826 replied:
@lvps It is for the `msgfmt` command in the setup process, should I remove it?

e-caste commented:
TODO before merge:
- add final echo showing the container has started and where to go (http://localhost:80) to find the development website
- run the container as daemon (to get back to the terminal once finished launching)

luco5826 commented:
@e-caste @lvps  I just updated the script, right now it tells the dev where the webserver is accessible at.

I also added a `launch.sh` script that checks for `sqlite3`, `docker` and `docker-compose` installation, creates the `weeehire.db` file and launches the container

lvps commented:
@luco5826 Looks good, but we still need to get xdebug to work somehow

e-caste commented:
@lvps Let’s make another issue and corresponding PR for that

lvps commented:
@e-caste Agreed
